bin/apisix

```diff
@@ -37,9 +43,11 @@ if [[ -e $OR_EXEC && "$OR_VER" =~ "1.19" ]]; then
     LUAJIT_BIN=$(${OR_EXEC} -V 2>&1 | grep prefix | grep -Eo 'prefix=(.*)/nginx\s+--' | grep -Eo '/.*/')luajit/bin/luajit
 
     # use the luajit of openresty
+    echo "$LUAJIT_BIN $APISIX_LUA $*"
```

Comment:
it only for debug, all right? please confirm

Reply:
This is used to remind user which program they are using.